__tests__/app.test.js

```diff
@@ -44,3 +44,33 @@ describe("GET /api/", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  it("GET an article object with specific id", () => {
+    const articleId = 2;
+    return request(app)
+      .get(`/api/articles/${articleId}`)
+      .expect(200)
+      .then((res) => {
+        const { article } = res.body;
+        expect(article).toMatchObject({
```

Comment:
I'm not expecting any data for this, I am expecting the actual data for article 2 in this case. It is your database, you as the designer would/could know what article 2 is with a cursory search. As this is integration testing I would expect the test to be specific to that request.